### PR TITLE
Create Wiki model, controllers and views

### PR DESCRIPTION
**Pull request by the-b-boss**
In this feature branch:
- generated a Wiki model
- generated a Wiki controller 
- created the following controller actions for wikis: index, show, new, create, edit, update, destroy
- created the following views for wikis: index, show, new, edit, destroy
- created a wiki form partial
- refactored new and edit views to render the form partial

### CODE REVIEW

**Comment by the-b-boss** [comment]
Happy that everything is working as expected - so I have merged into master.

File: app/views/wikis/destroy.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Wikis#destroy</h1>
```
Comment (bjblock) [inline app/views/wikis/destroy.html.erb:1]: is this file necessary?  does it ever get rendered?

Comment (the-b-boss) [inline app/views/wikis/destroy.html.erb:1]: Good point. Think it was generated when I used the Rails Generator and guess I didn't spot it. Will delete it in my current branch and merge.
